tests/platform_tests/thermal_control_test_helper.py

```diff
@@ -283,7 +283,11 @@ def restart_thermal_control_daemon(dut):
         'Restarting thermal control daemon on {}...'.format(dut.hostname))
     find_thermalctld_pid_cmd = 'docker exec -i pmon bash -c \'pgrep -f thermalctld\' | sort'
     output = dut.shell(find_thermalctld_pid_cmd)
-    assert output["rc"] == 0, "Run command '%s' failed" % find_thermalctld_pid_cmd
+
+    if dut.facts["asic_type"] == "vs" and output["rc"] == 0:
+        pytest.skip("Thermalctld doesn't support on vs testbed")
+    else:
+        assert output["rc"] == 0, "Run command '%s' failed" % find_thermalctld_pid_cmd
```

Comment:
@yutongzhang-microsoft if you prefer to use format() to concentrate string, do you mind keeping using a same format for whole script?

Reply:
Sure, I will keep the same format in this script